app/controllers/shared/api.php

```diff
@@ -238,6 +269,30 @@
     }
 
     $route = $utopia->match($request);
+
+    $useCache = $route->getLabel('cache', false);
+    if ($useCache) {
+        $data = $response->getPayload();
+        if (!empty($data)) {
+            $key = md5($request->getURI() . implode('*', $request->getParams()));
+            $cacheLog = $dbForProject->getDocument('cache', $key);
+            if ($cacheLog->isEmpty()) {
+                Authorization::skip(fn () => $dbForProject->createDocument('cache', new Document([
+                '$id' => $key,
+                'accessedAt' => \time(),
+                ])));
+            } elseif (date('Y/m/d', \time()) > date('Y/m/d', $cacheLog->getAttribute('accessedAt'))) {
```

Comment:
Are you sure this condition works? Seems like it compares strings rather than objects or integers.

Reply:
From what I see the return value is string so this condition shouldn't work well.
See here: https://www.php.net/manual/en/function.date.php